src/utils/toyactuator.tsx

```diff
@@ -0,0 +1,97 @@
+import { type GenericDeviceMessageAttributes, ActuatorType } from 'buttplug';
```

Comment:
Since the logic for toys has grown siginifcantly, it is no longer suitable to live in the utils folder.
Could you move these into their own toys folder, structured around this feature? (see e.g. the e621 folder).

Reply:
Moved these in [0df5bb6](https://github.com/clynamic/joi.how/pull/18/commits/0df5bb68442ac87bda65dc1928eb5e986e045542). Not sure if there's any further structuring we want to do? If you'd like me to split these files up further or move some other toy-related stuff into this folder, I'll do that. Let me know what you think.